dts/arm/st/f0/stm32f0.dtsi

```diff
@@ -41,6 +41,7 @@
 				compatible = "soc-nv-flash";
 				label = "FLASH_STM32";
 
+				erase-block-size = <1024>;
```

Comment:
STM32F0 does not have a uniform page size (eg. STM32F09x does have 2k pages - RM0091 table 5).

Reply:
Oups, missed that, txs!